src/routes/email/email.controller.ts

```diff
@@ -82,4 +84,22 @@ export class EmailController {
       code: verifyEmailDto.code,
     });
   }
+
+  @Delete('')
+  @UseGuards(
+    EmailRegistrationGuard,
```

Comment:
This guard verifies the signed message follows the pattern `email-register-*`. I think the expected message for an email deletion should follow another pattern, right? Something like `email-delete-*` I guess. 

(This would also affect tests on `email.controller.delete-email.spec.ts`)

Reply:
I've modified the guard to now accept varying prefixes in e70624d86bf9189b7a6d40453a925f2aa32eeba4.